onnxruntime/python/tools/symbolic_shape_infer.py

```diff
@@ -1142,7 +1142,30 @@ def _infer_Split(self, node):
     def _infer_SplitToSequence(self, node):
         self._infer_Split_Common(node, helper.make_sequence_value_info)
 
-    def _infer_Squeeze(self, node):
+    def _infer_Squeeze(self, node): 
+        input_shape = self._get_shape(node, 0)
+
+        # Depending on op-version 'axes' are provided as attribute or via 2nd input      
+        axes = get_attribute(node, 'axes')
+        if axes == None:
+            axes = self._try_get_value(node, 1)
+
+        if axes == None:
+            # No axes have been provided (neither via attribute nor via input).
+            # In this case the 'Shape' op should remove all axis with dimension 1.
+            # For symbolic dimensions we guess they are !=1.
```

Comment:
>For symbolic dimensions we guess they are !=1 [](http://example.com/codeflow?start=14&length=45)

This may not necessarily be consistent with runtime behavior. Maybe should have a warning at verbose level 1 at least.

Reply:
Added warning on assumption that symbolic input diensions are assumed to be != 1.